### DISCUSSION TITLE
HSL not matching RGB or Hex values

### DISCUSSION THREAD

**Discussion by lagudelo02** [primer/prism#58] [Q&A] [answered]
Hi there!
I wanted to use Primer Prism to generate a full palette based on some existing colors, but when I enter the HSL values, the color I obtain in Primer Prism is very different from the one I see in Figma. What I noticed is that the RGB and the HEX values don't match with the ones from Figma based on the same HSL values. Am I doing something wrong or is there an issue with how the values are converted? I'm attaching a screenshot for reference. 

<img width="794" alt="Screenshot 2022-11-24 at 17 22 59" src="https://user-images.githubusercontent.com/14839741/203830195-26612457-cf6e-4d49-b94a-fb8bdb1f3d20.png">

**Comment by lesliecdubs** [comment]
Hello @lagudelo02! 

@colebemis can you weigh in on this when you have a moment?

**Reply by lukasoppermann** [reply, marked as answer]
Hey @lagudelo02 ,

its a bit late, but maybe I can help.

In Figma you are using `HSL` but prism is using `HSLuv`. You can use an `HSLuv` plugin in Figma to get a more realistic representation of the values.

`HSLuv` is a different color space so it has some differences.

**Reply by lagudelo02** [reply]
Hey @lukasoppermann, thanks for getting back to me and for clarifying! 
Will check for some plugins in Figma.
Thanks!
